### ISSUE TITLE
Editor 2D. Dragging/Resizing/Rotating handles should move along with the object

### ISSUE THREAD

**Issue by wyattbiker** [godotengine/godot#22118]
**Godot version:**
3.1
Mac OS X, High Sierra

**Issue description:**
iIn 2D editor, dragging/resizing/rotating handles should move along with the object.

**Comment by groud** [comment]
What do you mean? They do.
Please provide a video or a gif of the problem.

**Comment by wyattbiker** [comment]
Here you go.

[movingbug.mov.zip](https://github.com/godotengine/godot/files/2386770/movingbug.mov.zip)

**Comment by groud** [comment]
Hmm, I don't know if it comes from the gif but your computer seems really slow.
I cannot reproduce the bug on my side, maybe it's OS dependant. If anyone on a Mac could confirm that would be good.

**Comment by wyattbiker** [comment]
It works fine with 3.0

**Comment by groud** [comment]
> It works fine with 3.0. Watch the .mov video as well.

Indeed, it looks better with the mov file. Still I cannot reproduce on linux. It would be interesting to bissect to find the bad commit.

**Comment by wyattbiker** [comment]
All of a sudden today it started working.  But now I cannot reproduce this problem and I have not updated repository. Not sure what changed. Maybe better to close this issue for now.

**Comment by groud** [comment]
Just in case, maybe this was due to the GLES2 / GLES3 settings? I had some problems with that on my side, so I wonder if GLES2 did not cause that.

**Comment by wyattbiker** [comment]
Well, it happened again. And good catch with GLES2 as soon as I tried selecting 3 from drop down top right, it fixed itself. But now I cannot reproduce it going back and forth. Also the list will not save GLES3 it always displays 2.

In addition the Project Settings does not preserve GLES3 but reverts to 2 when you quit all the wway out and come back. 

Also why do I have 4 GLES on drop down?

![screen shot 2018-09-19 at 11 53 50 am](https://user-images.githubusercontent.com/930478/45765340-d4be5500-bc02-11e8-9bb5-b201a39adce8.png)

Ill reopen.

**Comment by groud** [comment]
Maybe there's a weirdness in your project.godot file, could you check the entries for the GLES2/GLES3 selection ?

**Comment by akien-mga** [comment]
The bug with 4 options in the toggle is fixed already in the master branch, you should update your build.
